index.html

```diff
@@ -39,18 +39,36 @@ <h3 class="logo__title"><a href="#" class="link__logo">My Logo</a></h3>
         <i class="fas fa-bars fa-lg menu--humb"></i>
```

Comment:
- [ ] OPTIONAL
To improve the **user experience** it will be good practice to ensure that the _hamburger_ and the _close button_ are behaving as buttons when the mouse hovers on them. We can improve upon this aspect by adding the CSS attribute `cursor: pointer` to the _hamburger_ and the _close buttons on _mouse hover_ events. We can additionally change the color or opacity of these objects during the hover event.